kazoo/recipe/lock.py

```diff
@@ -267,6 +291,56 @@ def __exit__(self, exc_type, exc_value, traceback):
         self.release()
 
 
+def WLock(*args, **kwargs):
+    """Kazoo read-write lock (writer side).
+
+    Example usage:
+
+    .. code-block:: python
+
+        zk = KazooClient()
+        lock = WLock("/lockpath", "my-identifier")
```

Comment:
That shouldn't be the case (no global), probably just the docs are not right here.